src/sentry/integrations/opsgenie/actions/notification.py

```diff
@@ -16,7 +16,9 @@
 class OpsgenieNotifyTeamAction(IntegrationEventAction):
     id = "sentry.integrations.opsgenie.notify_action.OpsgenieNotifyTeamAction"
     form_cls = OpsgenieNotifyTeamForm
-    label = "Send a notification to Opsgenie account {account} and team {team}"
+    label = (
+        "Send a notification to Opsgenie account {account} and team {team} with priority {priority}"
```

Comment:
nit: IMO "P2 priority" sounds more proper than "priority P2" so i think the identifier should go before the word 'priority'. 🤷